kornia/feature/disk/unets/blocks.py

```diff
@@ -0,0 +1,76 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from kornia.core import Tensor
+
+from .utils import cut_to_match, size_is_pow2
+
+
+class TrivialUpsample(nn.Module):
```

Comment:
from kornia.core
```suggestion
class TrivialUpsample(Module):
```

same for the others